### PR TITLE
Update timer backend to allow only one timer per worker

### PR DESCRIPTION
**Pull request by lydiagu**

### CODE REVIEW

File: orchestra/tests/helpers/__init__.py [hunk 1]
```diff
@@ -73,3 +74,17 @@ def assertModelInstanceExists(self,
     def assertModelInstanceNotExists(self, model_class, lookup_attributes):
         with self.assertRaises(model_class.DoesNotExist):
             model_class.objects.get(**lookup_attributes)
+
+
+class OrchestraTestCase(OrchestraTestHelpersMixin, TestCase):
+    # NOTE(lydia): Mixin should go before TestCase because when mixin calls
```
Comment (lydiagu) [inline orchestra/tests/helpers/__init__.py:80]: @marcua @joshblum good thing we spent a while figuring out multiple inheritance in python! also http://nedbatchelder.com/blog/201210/multiple_inheritance_is_hard.html

Comment (lydiagu) [inline orchestra/tests/helpers/__init__.py:80]: what I find strange is that do people create mixins which must be inherited with another class?

File: orchestra/utils/time_tracking.py [hunk 2]
```diff
@@ -80,59 +81,65 @@ def save_time_entry(worker, task_id, time_entry_data):
...
+    Returns TaskTimer object associated with worker, creates one if does not
+    exist.
+    """
     try:
-        timer = assignment.timer
+        timer = worker.timer
     except TaskTimer.DoesNotExist:
-        timer = TaskTimer.objects.create(worker=worker,
-                                         assignment=assignment)
+        timer = TaskTimer.objects.create(worker=worker)
```
Comment (jrbotros) [inline orchestra/utils/time_tracking.py]: `get_or_create`?

**Comment by jrbotros** [comment]
looks great!

**Pull request merged**